packages/hdwallet-ledger/src/ledger.ts

```diff
@@ -13,7 +13,7 @@ import {
   translateScriptType
 } from './utils'
 
-export function isLedger (wallet: core.HDWallet): wallet is LedgerHDWallet {
+export function isLedger (wallet: any): wallet is LedgerHDWallet {
```

Comment:
should go the other way on this, IMO.... makes for better type safety in typescript clients.